core/src/test/java/org/elasticsearch/client/transport/TransportClientRetryIT.java

```diff
@@ -69,7 +69,7 @@ public void testRetry() throws IOException, ExecutionException, InterruptedExcep
                 if (randomBoolean()) {
                     clusterState = client.admin().cluster().state(clusterStateRequest).get().getState();
                 } else {
-                    PlainListenableActionFuture<ClusterStateResponse> future = new PlainListenableActionFuture<>(client.threadPool());
+                    PlainActionFuture<ClusterStateResponse> future = PlainActionFuture.newFuture();
```

Comment:
oops, I didn't realize I stole these changes from you. I happened to make exactly the same changes to these two transport tests. Sorry!